samples/bluetooth/peripheral_cgm/README.rst

```diff
@@ -0,0 +1,74 @@
+.. _peripheral_cgm:
+
+Bluetooth: Continuous Glucose Monitoring Service (CGMS)
+#######################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Peripheral Continuous Glucose Monitoring Service (CGMS) sample demonstrates the use of continuous glucose service.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+.. include:: /includes/tfm.txt
+
+Overview
+********
+
+The sample demonstrates a basic Bluetooth® Low Energy Peripheral role functionality that exposes the Continuous Glucose Monitoring GATT Service.
+Once it connects to a Central device, it generates dummy glucose measurement values.
+You can use it together with the Continuous Glucose module in the nRF Toolbox app.
```

Comment:
It may be worth to mention that we can also use the `nrf Connect` app.?

Reply:
Good idea. nrf connect also support interpretation of the data.